jetty-util/src/main/java/org/eclipse/jetty/util/thread/ScheduledExecutorScheduler.java

```diff
@@ -108,13 +126,15 @@ protected void doStop() throws Exception
     {
         scheduler.shutdownNow();
```

Comment:
Must not shutdown an external scheduler, so this line should be guarded with `if (name != null)`

Reply:
Done.